### PR TITLE
release python sdk

### PR DESCRIPTION
**Pull request by gaoning777**
It builds a python package that we can upload to the pypi.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/metadata/41)

### CODE REVIEW

**Comment by gaoning777** [comment]
prepare for releasing the sdk: https://github.com/kubeflow/metadata/issues/40

File: sdk/python/setup.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+setup(
+    name=NAME,
+    version=VERSION,
+    description='KubeFlow Metadata SDK',
+    author='google',
+    install_requires=REQUIRES,
+    packages=[
+        'kfmd',
+        'kfmd.swagger_client',
```
Comment (zhenghuiwang) [inline sdk/python/setup.py]: Should we check in the swagger client? (if so, we can do it later)

If we don't check in the generated code in the source repo, should we generate it before publishing the package?

Comment (gaoning777) [reply sdk/python/setup.py]: The current build.sh requires the swagger client to be generated before the kfmd package can be built. In this case, the generated swagger client is not checked in the source repo but the code will be in the kfmd package.

Comment (zhenghuiwang) [reply sdk/python/setup.py]: Sounds good. I'm leaning towards checking in the generated code under source control but we can do it later.

**Comment by gaoning777** [comment]
Would you mind also approve since I'm not in the owners list?

**Pull request merged**